lib/common/options.c

```diff
@@ -347,7 +365,9 @@ pcmk__env_option_enabled(const char *daemon, const char *option)
 {
     const char *value = pcmk__env_option(option);
 
-    return (value != NULL) && (crm_is_true(value) || strstr(value, daemon));
+    return (value != NULL)
+        && (crm_is_true(value)
+            || ((daemon != NULL) && (strstr(value, daemon) != NULL)));
```

Comment:
I'd like to add a regex check at some point, rather than a simple substring check. Allow comma or whitespace delimiters but make sure the daemon exists as a discrete "word".

Reply:
See pcmk__strcmp(..., pcmk__str_regex)

Reply:
Thanks! I did already find that one, just didn't want to add more clutter to this PR :)